test/parallel/test-no-stacktrace-version.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+// Flags: --no-stacktrace-version
+
+// This test ensures that the version is hidden at the end of stacktraces
+
+const common = require('../common');
+const assert = require('assert');
+const exec = require('child_process').exec;
+const fixtures = require('../common/fixtures');
+
+function errExec(script, callback) {
+  const cmd = `"${process.argv[0]}" --no-stacktrace-version "${fixtures.path(script)}"`;
+  return exec(cmd, (err, stdout, stderr) => {
+    // There was some error
+    assert.ok(err);
+
+    // More than one line of error output.
+    assert.ok(stderr.split('\n').length);
+
+    // Proxy the args for more tests.
+    callback(err, stdout, stderr);
```

Comment:
I copy/paste this function from https://github.com/nodejs/node/blob/fa1a842a08aa0be1cbc7b6a0e5517340c6768986/test/parallel/test-error-reporting.js#L28-L40

The only change is the `cmd` variable to pass the right CLI option, there should be a way to follow the DRY principle, how could we make it reusable, so we could use it in `test/parallel/test-no-stacktrace-version.js`, `test/parallel/test-stacktrace-version.js` and `test/parallel/test-error-reporting.js` ?